docs/reference/inference/put-inference.asciidoc

```diff
@@ -6,11 +6,11 @@ experimental[]
 
 Creates a model to perform an {infer} task.
 
-IMPORTANT: The {infer} APIs enable you to use certain services, such as built-in 
-{ml} models (ELSER, E5), models uploaded through Eland, Cohere, OpenAI, or 
-Hugging Face, in your cluster. For built-in models and models uploaded though 
-Eland, the {infer} APIs offer an alternative way to use and manage trained 
-models. However, if you do not plan to use the {infer} APIs to use these models 
+IMPORTANT: The {infer} APIs enable you to use certain services, such as built-in
+{ml} models (ELSER, E5), models uploaded through Eland, Cohere, OpenAI, or
+Hugging Face, in your cluster. For built-in models and models uploaded though
```

Comment:
Should we be referencing "cluster" here? I believe we are trying to get away from clusters in serverless. Since we are going with the "elasticsearch" naming, should we say "in your elasticsearch instance" here? Or will that confuse people who aren't using serverless?

Reply:
Perhaps removing the "cluster" wording from the docs is something that should happen as a separate initiative.